spec/search_builders/hyrax/admin_admin_set_member_search_builder_spec.rb

```diff
@@ -13,7 +13,17 @@
 
     it 'searches for valid work types' do
       expect(builder.filter_models(solr_params))
-        .to contain_exactly("{!terms f=has_model_ssim}Monograph,#{Hyrax.config.collection_class}")
+        .to contain_exactly(include("{!terms f=has_model_ssim}Monograph"))
+    end
+
+    it 'searches for collections indexed as ActiveFedora' do
+      expect(builder.filter_models(solr_params))
+        .to contain_exactly(include("Collection"))
+    end
+
+    it 'searches for collections indexed as valkyrie' do
+      expect(builder.filter_models(solr_params))
+        .to contain_exactly(include(Hyrax.config.collection_class.to_s))
     end
```

Comment:
An alternative that searches for all known collections...
```suggestion
    it 'searches for collections' do
      Hyrax.collection_classes do |klass|
        expect(builder.filter_models(solr_params))
          .to contain_exactly(include(klass.to_s))
      end
    end
```